src/components/scores/homework-cell.cjsx

```diff
@@ -15,14 +15,14 @@ HomeworkCell = React.createClass
   getScore: (isAccepted) ->
     {task} = @props
     if isAccepted
```

Comment:
Can `getScore` and `getProgress` ditch the `isAccepted` argument? and just return the `...on_time... + ...accepted_late...` sum?